### ISSUE TITLE
Feature suggestion: allow other truthy convention

### ISSUE THREAD

**Issue by roperto** [adrienverge/yamllint#150]
I'd like to use yes/no instead of true/false as this seem to be used more often in Ansible.

I can try to make a PR if the maintainer thinks it could be a nice addition and can review the code.

Any thoughs?

**Comment by adrienverge** [comment]
This sounds reasonable.

Before writing any code, how would you think the configuration should look like? Currently it's just `truthy: disable` or `truthy: enable` or `truthy: {}`.

**Comment by roperto** [comment]
What about a list of keywords that are valid?

Ie `['yes', 'no']` ?

That should not affect the currently used options.

**Comment by adrienverge** [comment]
More precisely, what keyword would you use? `allowed-values`?
```yaml
truthy:
  allowed-values: [yes, no]
```
or

```yaml
truthy:
  allowed-values: [on, off]
```
or

```yaml
truthy: enable  # that defaults to [true, false]
```
?

**Comment by roperto** [comment]
Sounds like a plan.

We probably need to quote the values as strings in the configuration do they don't evaluate to booleans, is that about right?

**Comment by adrienverge** [comment]
You're totally right!

**Comment by samdoran** [comment]
I would love to see this added. This would be very useful for controlling valid booleans values rather than just allowing all the flexible values or _only_ `true/false`.

**Comment by evitalis** [comment]
I would also like to vote on being able to use flexible values and match what ansible allows to be valid. It won't hurt anything already using true/false but allow ansible playbooks to be linted with `yamllint` as well as `ansible-lint`.

I could see this not being changed though as YAML is not dictated by ansible itself but it does seem that many people use both linters for their playbooks.

**Comment by Lirt** [comment]
I like key `allowed-values` as suggested by @adrienverge.

Having this option would make life of all `ansible` folks much easier. Did someone start to work on this?

**Comment by evitalis** [comment]
@adrienverge Any chance of this being resolved resolved soon? It's making double checking ansible playbooks very noisy. Yes `ansible-lint` is tailored more for that project but a second review is not a bad thing.

**Comment by adrienverge** [comment]
@evitalis this will be resolved soon, if someone starts working on it 🙂 Do you volunteer?

**Comment by Lirt** [comment]
> @evitalis this will be resolved soon, if someone starts working on it slightly_smiling_face Do you volunteer?

I'm in, I will try to create MR today.

**Comment by Lirt** [comment]
Please review https://github.com/adrienverge/yamllint/pull/181. 

There is slightly reduced code coverage even though I made tests for this, but I hope everything is OK. 

Can this be released as for example `1.15.1`?

**Comment by evitalis** [comment]
@Lirt Thanks for taking this on. I had not had time to revisit this to get some code down. I can try and help test hopefully this weekend though if needed.

**Comment by adrienverge** [comment]
It will be released in 1.16.
